plugin/adsk/scripts/mayaUsd_createStageFromFile.mel

```diff
@@ -117,7 +117,7 @@ global proc stageFromFile_UICommit(string $parent) {
 
 proc string doCreateStage(string $fileName) {
     // actually load the file
-    string $baseName = capitalizeString(`basenameEx $fileName`);
+    string $baseName = basenameEx($fileName);
```

Comment:
I removed the capitalization from the mayaUsd plugin only. The Pixar plugin still has the capital code.